pytorch_lightning/metrics/functional/classification.py

```diff
@@ -0,0 +1,401 @@
+from collections import Sequence
+from typing import Optional, Tuple, Callable
+from functools import wraps
+
+import torch
+
+from pytorch_lightning.metrics.functional.reduction import reduce
+
+
+def to_onehot(tensor: torch.Tensor,
```

Comment:
mind add docs to all these functions and it would be nice to have a doctests here @awaelchli @SkafteNicki ?

Reply:
I can probably add some docs this evening :)

Reply:
I have added a bit description to some of the functions now

Reply:
@SkafteNicki how have you added docs, as a suggestion or direct commit? I cannot see any change from today...

Reply:
Commit 4d1eb4c7d257d26d692e4b3f031f1790f97342f8, not sure if I did what you asked for :]